tests/backends/test_pyevm.py

```diff
@@ -41,88 +37,111 @@ def test_generate_custom_genesis_state(self):
         assert len(account_keys) == 10
 
         # Test the underlying state merging functionality
-        genesis_state = generate_genesis_state_for_keys(account_keys=account_keys, overrides=state_overrides)
+        genesis_state = generate_genesis_state_for_keys(
+            account_keys=account_keys, overrides=state_overrides
+        )
         assert len(genesis_state) == len(account_keys) == 10
         for _public_address, account_state in genesis_state.items():
-            assert account_state['balance'] == state_overrides['balance']
-            assert account_state['code'] == b''
+            assert account_state["balance"] == state_overrides["balance"]
+            assert account_state["code"] == b""
 
         # Only existing default genesis state keys can be overridden
         with pytest.raises(ValueError):
-            _invalid_genesis_state = generate_genesis_state_for_keys(account_keys=account_keys,
-                                                                     overrides=invalid_overrides)
+            _invalid_genesis_state = generate_genesis_state_for_keys(
+                account_keys=account_keys, overrides=invalid_overrides
+            )
 
         # Use staticmethod state overriding
-        genesis_state = PyEVMBackend._generate_genesis_state(overrides=state_overrides, num_accounts=3)
+        genesis_state = PyEVMBackend._generate_genesis_state(
+            overrides=state_overrides, num_accounts=3
+        )
         assert len(genesis_state) == 3
         for _public_address, account_state in genesis_state.items():
-            assert account_state['balance'] == state_overrides['balance']
-            assert account_state['code'] == b''
+            assert account_state["balance"] == state_overrides["balance"]
+            assert account_state["code"] == b""
```

Comment:
Not sure if this was an automated change but we tend to try and isolate these types of changes from real code changes.  No need to separate here but in future PR's try to split this up into a "cleanup" PR and a "fix things" PR

Reply:
Never a need to apologize for your code (though I'm sure you'll see me still do it plenty). :+1: 

Thanks for the contribution.